README.md

```diff
@@ -705,6 +705,11 @@ Or Messagebird credentials, which can be obtained in the [Dashboard](https://das
 - `SMS_MESSAGEBIRD_ACCESS_KEY` - your Messagebird access key
 - `SMS_MESSAGEBIRD_ORIGINATOR` - SMS sender (your Messagebird phone number with + or company name)
 
+Or SendBlue credentials, available at the [Dashboard](https://sendblue.co/dashboard/api)
+
+- `SMS_SENDBLUE_KEY_ID` - called "API Key" (first field) on the dashboard
+- `SMS_SENDBLUE_SECRET_KEY` - called "API Secret" (second field) on the dashboard
```

Comment:
Can you call the config variables the same as they're called in the SendBlue API?

So `SMS_SENDBLUE_API_KEY` and `SMS_SENDBLUE_API_SECRET`. This should avoid any confusion.